recipes/python-bioext/build.sh

```diff
@@ -9,3 +9,15 @@ fi
 export FREETYPE2_ROOT=$PREFIX
 
 $PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+for CHANGE in "activate" "deactivate"
+do
+    mkdir -p "${PREFIX}/etc/conda/${CHANGE}.d"
+    cp "${RECIPE_DIR}/${CHANGE}.sh" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
+done
+
+# Symlink reference data to /usr/local/share, since the site-packages path may vary depending on python version
+mkdir -p "${PREFIX}/usr/local/share/bealign_references"
```

Comment:
What about `$PREFIX/share/$PKG_NAME/bealign_references` ? It seems more standard. Also, I'd define a `TARGET` environment variable and reuse it in `mkdir` and the `ln` commands below.

Reply:
Ooooh, that's a good idea, grazie.